README.md

```diff
@@ -46,6 +46,28 @@ The `-p` option can be used to skip loading any plugin modules.
 The `-s` option can be used to only parse PELs if they are for serviceable
 errors, while the `-n` option will only parse PELs for non-serviceable ones.
 
+### Installation
+
+If `setuptools` is missing, use this following command to install it.
```

Comment:
This only works on ubuntu/debian.  How about something like:  Install it from your distro, for example ...